### ISSUE TITLE
Remove dead code in helpers

### ISSUE THREAD

**Issue by d4hines** [marigold-dev/deku#225]
After #172 , we should remove some dead code in:
- helpers/yojson_ext.rei
- helpers/list_ext.re

See comments:
https://github.com/marigold-dev/deku/pull/172#discussion_r723614998

https://github.com/marigold-dev/deku/pull/172#discussion_r723546884

**Comment by d4hines** [comment]
No longer relevant.
